src/github/api/write.rs

```diff
@@ -429,7 +429,9 @@ impl GitHubWrite {
                 {id_field}: $id, 
                 pattern: $pattern, 
                 requiresStatusChecks: true, 
-                requiredStatusCheckContexts: $contexts, 
+                requiredStatusCheckContexts: $contexts,
+                # Disable 'Require branch to be up-to-date before merging'
+                requiresStrictStatusChecks: false,
```

Comment:
Do you by chance have any data on how often this might be set to `true` in the organization? Should we make this configurable through the team repo?

Reply:
I don't have the data right now, but should be easy to get with your or Mark's help. Last time I asked about this on #t-infra, some people said that the config is quite annoying, and I have to agree.